de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/nio/writer/debug/DebugArgumentDumping.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.nio.writer.debug;
+
+import java.io.PrintWriter;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.uima.fit.util.JCasUtil;
+import org.apache.uima.jcas.JCas;
+
+import de.tudarmstadt.ukp.dkpro.argumentation.types.ArgumentComponent;
+import de.tudarmstadt.ukp.dkpro.argumentation.types.ArgumentRelation;
+import de.tudarmstadt.ukp.dkpro.argumentation.types.ArgumentUnit;
+import de.tudarmstadt.ukp.dkpro.argumentation.types.ArgumentUnitUtils;
+import de.tudarmstadt.ukp.dkpro.argumentation.types.Claim;
+import de.tudarmstadt.ukp.dkpro.core.api.metadata.type.DocumentMetaData;
+
+/**
+ * @author <a href="mailto:shore@ukp.informatik.tu-darmstadt.de">Todd Shore</a>
+ * @since Apr 29, 2016
+ *
+ */
+final class DebugArgumentDumping {
```

Comment:
I think there is already such a class - de.tudarmstadt.ukp.dkpro.argumentation.io.writer.ArgumentDumpWriter